.github/workflows/scenario-test.yaml

```
@@ -0,0 +1,150 @@
+name: scenario-test
+on:
+  workflow_dispatch:
```

We would like to run this periodically at the beginning. 
```suggestion
  schedule:
    - cron: 0 12 * * *
  workflow_dispatch:
```